src/enumerator/arithmetic_sequence_object.cpp

```diff
@@ -0,0 +1,261 @@
+#include "natalie/enumerator/arithmetic_sequence_object.hpp"
+#include "natalie.hpp"
+
+namespace Natalie::Enumerator {
+ArithmeticSequenceObject::ArithmeticSequenceObject(Env *env, Origin origin, Value begin, Value end, Value step, bool exclude_end)
+    : ArithmeticSequenceObject {} {
+    m_origin = origin;
+    m_begin = begin;
+    m_end = end;
+    m_exclude_end = exclude_end;
+    m_step = step;
```

Comment:
Can we use the member initializer syntax for these members, like we do in the other classes?

```suggestion
    : Object { fetch_nested_const({ "Enumerator"_s, "ArithmeticSequence"_s }) }
    , m_origin { origin }
    , m_begin { begin }
    , m_end { end }
    , m_exclude_end { exclude_end }
    , m_step { step } {
```